admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -146,7 +146,7 @@
           <% end %>
         >
           <% @data.columns.each do |column| %>
-            <%= render_data_cell(column, row) %>
+            <%= render_data_cell(column, row, @data.url&.call(row)) %>
```

Comment:
Can you explain what this change does? I can try to figure out, but I am not yet very fluent in this part of Solidus yet.